lib/logstash/outputs/elasticsearch.rb

```diff
@@ -133,6 +133,11 @@ class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
   # a timeout occurs, the request will be retried.
   config :timeout, :validate => :number
 
+  # Set the Elasticsearch errors that you don't want to log. By default we log 
+  # every error. A useful example is when you want to skip all 409 errors
+  # which are `document_already_exists_exception`
+  config :silence_failures, :validate => :array, :default  => [""]
```

Comment:
`config :silence_failures, :validate => :number, :list => true`

I am new to this list type, shouldn't this be a list of strings? if so it's `:validate => :string`? and how do you set the defaults?